docs/api/menu.md

```diff
@@ -17,7 +17,8 @@ The `menu` class has the following static methods:
 * `menu` Menu
 
 Sets `menu` as the application menu on macOS. On Windows and Linux, the `menu`
-will be set as each window's top menu.
+will be set as each window's top menu. Setting it to `null` will remove the
+menu bar.
```

Comment:
> Setting it to `null` will remove the menu bar.

This isn't true on macOS so maybe it should be tied into the previous sentence that mentions Windows and Linux.